.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules
```

Comment:
If you're using Git, adding the `/node_modules` file in the .gitignore will mean git/Github won't try and upload it to your repo. This is generally a good minimum to have in any node project.